### PR TITLE
Resnet

### PR DESCRIPTION
**Pull request by ShamieCC**

### CODE REVIEW

**Comment by sezan92** [comment]
@ShamieCC I am unable to review until next weekend. can you assign new members ? @emre570 for example

File: shamie-Resnet.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Introduction:
+Observed Problem: Degradation Problem
+As the depth of the neural network deepens, the accuracy starts to saturate.
+
+The pre-existing perspectives of deeper layers were always thought of as effective because the perfomance of neural networks improved a lot by deepening the layers,
+since as the layers deepened, the level of extracted features could be further enrinched such as was seen with VGG16 and VGG19.
+A question arised: "Is learning networks as easy as stacking more layers"?
+The obstacle to answering this question, "gradient vanishing problem", was adressed by Normalised initializations and intermediate normalization layers.
+However, the degradation problem was observed, as the depth of the neural network deepened, the accuracy gets saturated and degrades rapidly. An experiment to explore the problem
+was conducted, comparing a shalloww model and a deep model. The training error and the test error of the Deeper model increased than with the shallow model. This was not because of overfitting but because the training error increased when the network deepened.
```
Comment (sezan92) [inline shamie-Resnet.mdx]:
```suggestion
was conducted, comparing a shallow model and a deep model. The training error and the test error of the Deeper model increased than with the shallow model. This was not because of overfitting but because the training error increased when the network deepened.

```

Comment (sezan92) [reply shamie-Resnet.mdx]: sorry I got confused. Can you rephrase?

File: shamie-Resnet.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Experiments with Resnet Showed that when used, even extremly deep neural networks, were easy to optimize with Resnet and the deeper the network, the higher the accuracy,compared to Plain-net which showed increased training error when the networks deepened.
```
Comment (sezan92) [inline shamie-Resnet.mdx]:
```suggestion
Empirical results showed that the Resnet outperformed even with the extremely deeper architectures . They were easier to optimize with higher accuracy compared to the previous architectures.
```

**Review by sezan92 — CHANGES_REQUESTED** [review]
Some rephrasing is needed. I suggest to use Grammarly to rephrase. Free version is okay.

File: shamie-Resnet.mdx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+Resnet Paper Summary:
+
+Introduction:
+Observed Problem: Degradation Problem
+As the depth of the neural network deepens, the accuracy starts to saturate.
+
+The pre-existing perspectives of deeper layers were always thought of as effective because the perfomance of neural networks improved a lot by deepening the layers,
```
Comment (sitammeur) [inline shamie-Resnet.mdx]:
```suggestion
The pre-existing perspectives of deeper layers was always thought of as effective because the performance of neural networks improved a lot by deepening the layers.

```

File: shamie-Resnet.mdx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+Resnet Paper Summary:
+
+Introduction:
+Observed Problem: Degradation Problem
+As the depth of the neural network deepens, the accuracy starts to saturate.
+
+The pre-existing perspectives of deeper layers were always thought of as effective because the perfomance of neural networks improved a lot by deepening the layers,
+since as the layers deepened, the level of extracted features could be further enrinched such as was seen with VGG16 and VGG19.
```
Comment (sitammeur) [inline shamie-Resnet.mdx]:
```suggestion
As the layers deepened, the extracted features could be further enriched, such as seen with VGG16 and VGG19.

```

File: shamie-Resnet.mdx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Resnet Paper Summary:
+
+Introduction:
+Observed Problem: Degradation Problem
+As the depth of the neural network deepens, the accuracy starts to saturate.
+
+The pre-existing perspectives of deeper layers were always thought of as effective because the perfomance of neural networks improved a lot by deepening the layers,
+since as the layers deepened, the level of extracted features could be further enrinched such as was seen with VGG16 and VGG19.
+A question arised: "Is learning networks as easy as stacking more layers"?
```
Comment (sitammeur) [inline shamie-Resnet.mdx:11]:
```suggestion
A question was raised: "Is learning networks as easy as stacking more layers"?

```

File: shamie-Resnet.mdx [hunk 6]
```diff
@@ -0,0 +1,20 @@
+Resnet Paper Summary:
+
+Introduction:
+Observed Problem: Degradation Problem
+As the depth of the neural network deepens, the accuracy starts to saturate.
+
+The pre-existing perspectives of deeper layers were always thought of as effective because the perfomance of neural networks improved a lot by deepening the layers,
+since as the layers deepened, the level of extracted features could be further enrinched such as was seen with VGG16 and VGG19.
+A question arised: "Is learning networks as easy as stacking more layers"?
+The obstacle to answering this question, "gradient vanishing problem", was adressed by Normalised initializations and intermediate normalization layers.
```
Comment (sitammeur) [inline shamie-Resnet.mdx]:
```suggestion
The obstacle to answering this question, the "gradient vanishing problem", was addressed by Normalised initializations and intermediate normalization layers.

```

**Review by sitammeur — CHANGES_REQUESTED** [review]
Overall, as we discussed earlier, it was good. These are some changes that are needed, as mentioned in the review.

**Comment by sitammeur** [comment]
> Some rephrasing is needed. I suggest to use Grammarly to rephrase. Free version is okay.

@ShamieCC  I checked with Grammarly and proposed these changes. Still, it's better if you check it yourself!

**Comment by ShamieCC** [comment]
> Some rephrasing is needed. I suggest to use Grammarly to rephrase. Free version is okay.

I have installed Grammarly now and will use it going forward thank you.

**Comment by ShamieCC** [comment]
> > Some rephrasing is needed. I suggest to use Grammarly to rephrase. Free version is okay.
> 
> @ShamieCC I checked with Grammarly and proposed these changes. Still, it's better if you check it yourself!

**Comment by ShamieCC** [comment]
Thank you for the proposed changes, I have committed some of them. I will use Grammarly from now on.
